services/order.py

```diff
@@ -0,0 +1,23 @@
+from django.db import transaction
+from db.models import Order, Ticket, MovieSession, User
+
+def create_order(tickets, username, date=None):
+    user = User.objects.filter(username=username).first()
+    if not user:
+        return None
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user, created_at=date) if date else Order.objects.create(user=user)
+        for ticket in tickets:
+            Ticket.objects.create(
+                movie_session=MovieSession.objects.get(id=ticket["movie_session"]),

```

Comment:
Consider adding error handling for the `MovieSession.objects.get(id=ticket["movie_session"])` call to manage cases where the `MovieSession` does not exist, which would raise a `DoesNotExist` exception.